channels/pkg/channels/addon.go

```diff
@@ -152,3 +164,36 @@ func (a *Addon) EnsureUpdated(ctx context.Context, k8sClient kubernetes.Interfac
 
 	return required, nil
 }
+
+func (a *Addon) AddNeedsUpdateLabel(ctx context.Context, k8sClient kubernetes.Interface) error {
+	klog.Infof("addon %v wants to roll %v nodes", a.Name, a.Spec.NeedUpdate)
```

Comment:
```suggestion
	klog.Infof("addon %v wants to update %v nodes", a.Name, a.Spec.NeedUpdate)
```

Reply:
This appears to have been dropped without change or comment.

Reply:
Forgot to pull in this change before force pushing ...

Reply:
Figured it was something like that.